client/src/store/actions/adminLoginActions.js

```diff
@@ -0,0 +1,3 @@
+const SET_LOGIN = 'SET_LOGIN';
```

Comment:
I know the merit of this however we should be consistent and this hasn't been discussed before, so hasn't been used in any other reducers.

Reply:
Refer to Michaels tech demo ;D